cmd/cluster-olm-operator/main.go

```diff
@@ -105,13 +106,28 @@ func runOperator(ctx context.Context, cc *controllercmd.ControllerContext) error
 		deploymentControllerList = append(deploymentControllerList, controller)
 	}
 
+	operatorImageVersion := status.VersionForOperatorFromEnv()
+	nextOCPMinorVersion, err := utils.GetNextOCPMinorVersion(operatorImageVersion)
+	if err != nil {
+		return err
+	}
+
 	upgradeableConditionController := controller.NewStaticUpgradeableConditionController(
 		"OLMStaticUpgradeableConditionController",
 		cl.OperatorClient,
 		cc.EventRecorder.ForComponent("OLMStaticUpgradeableConditionController"),
 		controllerNames,
 	)
 
+	incompatibleOperatorController := controller.NewIncompatibleOperatorController(
+		"OLMincompatibleOperatorController",
```

Comment:
```suggestion
		"OLMIncompatibleOperatorController",
```

I know `OLMI` is a bit jarring since we're combining an acronym with CamelCase, but that is the convention (e.g. `IPAddress`, `TCPAddr`, etc.)